src/world.rs

```diff
@@ -201,6 +259,19 @@ impl World {
 }
 
 impl World {
+    /// Check if a certain component type is present for an [Entity](struct.Entity.html).
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use rs_ecs::*;
+    /// let mut world = World::new();
+    ///
+    /// let entity = world.alloc();
+    /// world.insert(entity, (42_u32, true));
+    ///
+    /// let has_u32 = world.contains::<u32>(entity);
```

Comment:
To make the doc tests more powerful, this should actually be an assertion, e.g. `assert!(world.contains::<u32>(entity));`.